bom/runtime/pom.xml

```diff
@@ -69,6 +69,8 @@
         <graal-sdk.version>19.2.0.1</graal-sdk.version>
         <gizmo.version>1.0.0.Alpha8</gizmo.version>
         <jackson.version>2.9.10</jackson.version>
+        <!-- Fix a databing version higher that the one from the BOM for CVE vulnerabilities: CVE-2019-16942, CVE-2019-16943, CVE-2019-17531 -->
+        <jackson-databind.version>2.9.10.1</jackson-databind.version>
```

Comment:
Adding another property and dependency is not need, the BOM is used in Quarkus, so the BOM  should be updated.

The Jackson BOM version corresponding to this fix is `2.9.10.20191020` so is enough to change
```xml
<jackson.version>2.9.10.20191020</jackson.version>
```